__tests__/app.test.js

```diff
@@ -372,3 +372,35 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("returns 200 status code and an array of users", () => {
+    return request(app)
+      .get("/api/users")
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body.users.length).toBe(4);
+        response.body.users.forEach((user) => {
+          expect(user).toHaveProperty("username");
+          expect(user).toHaveProperty("name");
+          expect(user).toHaveProperty("avatar_url");
```

Comment:
Lines `384 - 386` are not necessary as they are covered by lines `390 - 392` where you are making a detailed assertion for an object with those properties